18 Runners/README.md

```diff
@@ -0,0 +1,27 @@
+# Runners
+
+**Description:**
+
+This Koan makes use of Runners, which enables you to use your own infrastructure for running jobs. This means you will be able to build and test on a wider variety of architectures, as well as have additional control over the environment.
+
+What key aspects are important to note?
+
+**Goals:**
+
+- Carry out all pre-requisites to running tasks on a runner (authentication etc)
+- Create your runner using the Docker method, and successfully start the runner.
```

Comment:
We should likely note that the Docker container can be ran locally, essentially executing the Runner job locally, so they know they don't need to get too fancy beyond spinning up the container on their work machine.